lucien-ios/ComicDetailScreen/ComicDetailViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ComicDetailViewModel.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/24/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+import RxCocoa
+import RxSwift
+
+final class ComicDetailViewModel {
+
+    private var dashboardComicUser: DashboardComicUser
+
+    let comicImage = Variable(UIImage())
+    let comicTitle = Variable("-")
```

Comment:
What happens if we decide to change the placeholder string? You'll have to go through and change every string manually. Instead, save it as a constant and reference that here